src/object/authenticate_cache.cpp

```diff
@@ -408,6 +431,26 @@ authenticate_cache::get_cache_bits (SM_CLASS *sm_class)
   return &cache->data[cache_index];
 }
 
+unsigned int *
+authenticate_cache::get_procedure_cache_bits (MOP proc_mop)
+{
+  std::vector<unsigned int> *bits = nullptr;
+
+  auto it = procedure_cache.find (proc_mop);
+  if (it == procedure_cache.end ())
+    {
+      bits = new std::vector<unsigned int> (cache_max, AU_CACHE_INVALID);
+      procedure_cache[proc_mop] = bits;
+    }
+  else
+    {
+      bits = it->second;
+    }
+
+  bits->resize (cache_max, AU_CACHE_INVALID);
+  return &bits->at (cache_index);
```

Comment:
```suggestion
  bits->resize (cache_max, AU_CACHE_INVALID);
  return &(*bits)[cache_index];
```
1. authenticate_cache 로직에 의해 cache_index < cache_max가 보장되고 있습니다. 또 크기가 증가될 경우에만 초기화하며 늘어나므로, 성능 문제는 크지 않습니다.
2. bits->resize ()에 의해 out of range에 대한 방어코드는 만족됩니다. 

3. at () 에 의한 주소값을 반환하기 보다 []로 가져오도록 변경했습니다.